.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
The image line could of course be `consul:0.9.0` if we want to use the official upstream, we had to build our own as `0.9.11` official hasn't been released on dockerhub